content/en/cloud_cost_management/google_cloud.md

```diff
@@ -95,17 +96,37 @@ The following APIs and permissions enable Datadog to access your Detailed Usage
 
   **Note:** This can be a custom role, or you can use the existing Google Cloud role `roles/bigquery.dataEditor`.
 
-#### (Optional) Configure required cross-project service authorization:
+### Create or select a Google Cloud Storage bucket
+Use an existing Google Cloud Storage bucket or create a new one.
+Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to the selected bucket, where it will be prefixed with `datadog_cloud_cost_detailed_usage_export`.
+
+**Note:** The bucket [must be co-located][9] with the BigQuery export dataset.
+
+#### Configure bucket access
+[Add the service account as a principal on the GCS bucket resource][6]:
+1. Select your bucket in the Google Cloud console (go to the Cloud Storage Buckets page).
+2. Select the Permissions tab near the top of the page and click the 'Grant access' button.
+3. In the New principals field, enter the service account.
+4. Assign a role with the following permissions:
+   * `storage.buckets.get`
+   * `storage.objects.create`
+   * `storage.objects.delete`
+   * `storage.objects.get`
+   * `storage.objects.list`
+
+  **Note:** This can be a custom role, or you can use the existing Google Cloud roles `roles/storage.legacyObjectReader` and `roles/storage.legacyBucketWriter`.
+
+### (Optional) Configure cross-project service authorization:
 If your integrated Service Account exists in a different Google Cloud Platform project than your billing export dataset, you need to [grant cross-project service account authorization][10]:
 
-[Trigger service agent creation][11] using the following values:
+1. Trigger the service agent creation by following the [official documentation][11] using the following values:
   * ENDPOINT: `bigquerydatatransfer.googleapis.com`
   * RESOURCE_TYPE: `project`
   * RESOURCE_ID: export dataset project
```

Comment:
Debating if we should indent these 3 bullet points to improve readability 🧐 

<img width="776" alt="Screenshot 2024-02-09 at 13 56 08" src="https://github.com/DataDog/documentation/assets/14238324/54d202a9-bd0b-46d4-89c7-c65f2d216618">